kaggle_cli/download.py

```diff
@@ -46,13 +48,35 @@ def download_file(self, browser, url):
         self.app.stdout.write('downloading %s\n' % url)
         local_filename = url.split('/')[-1]
         headers = {}
+        file_size = 0
+        total_size = 0
+        bar = progressbar.ProgressBar()
 
         if os.path.isfile(local_filename):
             file_size = os.path.getsize(local_filename)
             headers['Range'] = 'bytes={}-'.format(file_size)
 
+        with closing(browser.get(url, stream=True)) as stream_orig:
+            total_size = int(stream_orig.headers['Content-Length'].strip())
+
+        self.bytes = file_size
+
         stream = browser.get(url, stream=True, headers=headers)
 
+        widgets = [local_filename,' ',progressbar.Percentage(),' ', progressbar.Bar(marker="#"),
```

Comment:
following the PEP8 conventions https://pep8.readthedocs.io/en/1.4.6/intro.html#error-codes, the a space should be attached after the comma.